### ISSUE TITLE
Model training Loss graph

### ISSUE THREAD

**Issue by Ugness** [richzhang/colorization-pytorch#3]
![newplot](https://user-images.githubusercontent.com/33034982/46448393-90fe4a80-c7c1-11e8-8f58-38b2d1c89c39.png)

I am training the model with ILSVRC2012 training set and same options as your implementation and my loss graph looks like above.  
I am afraid that my model's loss reduces correctly. Can you check this loss graph or share your loss graph?  
Thanks.

**Comment by richzhang** [comment]
Thanks for the message. I'll look into it in the next week or so.

**Comment by richzhang** [comment]
My apologies for the delay. I'm caught up with a few things and will have to look into this more after CVPR deadline (mid-Nov).

**Comment by Ugness** [comment]
Thanks.

**Comment by Samir55** [comment]
Hello author,
I have the same problem above, what is the solution? 
Thanks in advance.

**Comment by richzhang** [comment]
I believe I did not do the initialization properly. I am looking into it. Thanks for your patience!

**Comment by richzhang** [comment]
The problem was with the initialization. In the original paper, I used [k-means](https://github.com/philkr/magic_init) initialization, but didn't have the code for pytorch.

I just pushed a change. If you rerun `python make_ilsvrc_dataset.py`, it will make a directory `dataset/ilsvrc2012/train_small/`, which contains 10 imagenet classes. Then rerun `bash scripts/train_siggraph.sh` again. It will first run 100 epochs on this small training set as "initialization", before training on the full trianing set.

Thanks for your patience.
